extensions/positron-assistant/src/utils.ts

```diff
@@ -212,46 +232,64 @@ export function markBedrockCacheBreakpoint(message: ai.CoreMessage): ai.CoreMess
 function convertToolResultToAiMessageExperimentalContent(
```

Comment:
This function name (and others with `ExperimentalContent`) is a bit of a misnomer, now that the output message no longer has an `experimental_content` field.